gnirehtet.cmd

```diff
@@ -0,0 +1,150 @@
+@ECHO OFF
+CALL :locate_file ADB %~dp0adb.exe "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
```

Comment:
> In windows, unlike Linux, adb and Java doesn't automatically added to the Path environment variable.

Actually, on linux too, if `adb` is installed manually (i.e. not from the distribution packages) in a random directory, this directory must be added to `$PATH`.

> They only way to add these is manually, which is isn't so easy.

But it's the only way to call `adb.exe` in a terminal from anywhere, isn't it?

> I'm using the default location of the binaries of adb and Java which I believe is more likely to be

I'm ok with this.

However, may I suggest an alternative: instead of locating `adb.exe` manually, we could add the default location to the `%PATH%` locally so that any call to `adb.exe` will _also_ look at this location:

```batch
SET PATH=%PATH%;%LOCALAPPDATA%\Android\sdk\platform-tools
CALL adb.exe shell …
```

What do you think?